### ISSUE TITLE
WSDistance between two identical spectra

### ISSUE THREAD

**Issue by lgatto** [mciach/masserstein#12]
I tried to compute the difference between two identical spectra
```
$ python ./masserstein/WSDistance.py spectrum1.txt spectrum1.txt
Spectrum 1: spectrum1.txt
Spectrum 2: spectrum1.txt
Intensity cutoff: 0.99

Wasserstein distance:
0.10958221578852129
```
I am a bit surprised that the distance isn't 0.

**Comment by michalsta** [comment]
Hi

Could you provide the spectrum (if it's not secret data or something)? Will make it easier to reproduce ;)

**Comment by lgatto** [comment]
I used yours in `examples/protein_spectrum.txt`:

```
$ cp examples/protein_spectrum.txt ./spectrum1.txt
$ python ./masserstein/WSDistance.py spectrum1.txt spectrum1.txt
Spectrum 1: spectrum1.txt
Spectrum 2: spectrum1.txt
Intensity cutoff: 0.99

Wasserstein distance:
0.10958221578852129
```

**Comment by mciach** [comment]
This is because, when this script was first created 2 years ago, we assumed that the main use case would be to compare an experimental with a simulated spectrum. This was before we developed the automatic denoising procedure, so the distance would be very sensitive to any noise. The quick-and-dirty solution was to remove a small fraction of the signal from the first spectrum. The amount removed can be controlled by parameters of this script, described in the help message. 

In the near future this script will probably be removed anyway, because we started to develop this project more as a Python package rather than a set of console scripts
